.gitlab-ci.yml

```diff
@@ -5,17 +5,19 @@ stages:
 
 # This folder is cached between builds
 # http://docs.gitlab.com/ce/ci/yaml/README.html#cache
-cache:
-  key: ${CI_COMMIT_REF_SLUG}
-  paths:
-    - node_modules/
 
 build_site:
   image: node:10.16.0
   stage: build
   script:
     - npm install
+    - npm run type-check
```

Comment:
these two should probably be in their own `built_lint` job. That way, you can only run linting on merge request and not waste it on merges to master and such.

Reply:
Good call. I'll change that